cesium/internal/domain/writer.go

```diff
@@ -120,6 +120,12 @@ type Writer struct {
 	// fileKey represents the key of the file written to by the writer. One can convert it
 	// to a filename via the fileKeyToName function.
 	fileKey uint16
```

Comment:
This means we can have a maximum of 65,000 files. Is that enough? Maybe. I don't know.

Reply:
Note for future: 65000 Files of 1gb each ~= 65TB / channel. this should be fine.